### PR TITLE
chore: Migrate /team to camelCase

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/user/UserRepository.js [hunk 1]
```diff
@@ -1065,7 +1065,7 @@ z.user.UserRepository = class UserRepository {
   }
 
   map_guest_status(user_ets = this.users()) {
-    const team_members = this.team_members();
+    const team_members = this.teamMembers();
```
Comment (lipis) [inline app/script/user/UserRepository.js:1068]: maybe the `const` as well?

Comment (gregor) [reply app/script/user/UserRepository.js:1068]: Will happen when I refactor /users

Comment (lipis) [reply app/script/user/UserRepository.js:1068]: right..

Comment (lipis) [reply app/script/user/UserRepository.js:1068]: just saw that it's in the `/user` makes sense..

**Pull request merged**